js/build.js

```diff
@@ -0,0 +1,344 @@
+Fliplet.Widget.instance({
+  name: 'file-list',
+  displayName: 'File list',
+  icon: 'fa-exchange',
+  render: {
+    dependencies: [],
+    template: [
+      '<div data-view="content"></div>',
+    ].join(''),
+    ready: async function () {
+      var thisy = this;
+      const populateFileList = (entryID = Fliplet.Navigate.query.dataSourceEntryId, dsID = thisy.fields.dataSource) => {
+        var dataSourceId = dsID;
+        var dataSourceEntryId = entryID;
+        var columnName = thisy.fields.columnName; // TODO missing columName for list repeator
+        var type = thisy.fields.type;
+
+        if (!navigator.onLine) {
+          return Promise.reject('Please connect device to the internet')
```

Comment:
Missing semicolon. Any reason why this returns a promise as opposed to displaying a toast message like the next errors?

Reply:
Fixed.